sphinx_needs/external_needs.py

```diff
@@ -152,12 +158,11 @@ def load_external_needs(app: Sphinx, env: BuildEnvironment, docname: str) -> Non
                 )
 
             need_params["content"] = need["description"]
-            need_params["links"] = need.get("links", [])
-            need_params["tags"] = ",".join(need.get("tags", []))
-            need_params["status"] = need.get("status")
-            need_params["constraints"] = need.get("constraints", [])
 
-            del need_params["description"]
+            # Remove unknown options, as they may be defined in source system, but not in this sphinx project
+            for option in list(need_params):
+                if option not in known_options:
+                    del need_params[option]
```

Comment:
I feel this is worth a log message (type info) to the user.

Reply:
Eurgh, I'm worried this may lead to a tonne of "spurious" logging in existing projects if we do that, so want to consider if we would have a config or something to turn it off.
I would leave for a separate PR